src/cli/validator/parser/env_filter.rs

```diff
@@ -0,0 +1,76 @@
+use clap::builder::{NonEmptyStringValueParser, PossibleValue, TypedValueParser};
+use clap::error::{ContextKind, ContextValue, ErrorKind};
+use tracing::Level;
+use tracing_subscriber::EnvFilter;
+
+#[derive(Debug)]
+pub struct CustomEnvFilter(pub EnvFilter);
+
+impl Clone for CustomEnvFilter {
+	fn clone(&self) -> Self {
+		Self {
+			0: EnvFilter::builder().parse(self.0.to_string()).unwrap(),
```

Comment:
I hate bikeshedding like this but this strikes me as unusual. How about this instead?

```rust
let filter = EnvFilter::builder().parse(self.0.to_string()).unwrap();
Self(filter)
```